api/krusty/generatormergeandreplace_test.go

```diff
@@ -455,6 +455,165 @@ metadata:
 `)
 }
 
+func TestMergeAndReplaceDisableNameSuffixHashGenerators(t *testing.T) {
+	th := kusttest_test.MakeHarness(t)
+	makeBaseWithGenerators(th)
```

Comment:
nit: most of what this helper includes isn't really relevant to the test case. I'd prefer to use a Kustomization focussed on the relevant generators, to make it easier to understand the test case, but this isn't a blocker.

Reply:
Thanks for the feedback, that's a great point! I will add a commit to address this in the next few days.